lib/paper_trail.rb

```diff
@@ -174,21 +204,20 @@ def transaction?
       ::ActiveRecord::Base.connection.open_transactions.positive?
     end
 
-    # @api public
+    # @api private
```

Comment:
Is this intentional?  Again I'm not sure how you like to do this maintenance-wise, but I think that maybe we should add a deprecation notice that this is no longer a public method.

Reply:
Yes, intentional. I'll be changing the deprecation warning to say "deprecated without replacement".